schema/composition/animation.json

```diff
@@ -10,6 +10,12 @@
         {
             "type": "object",
             "properties": {
+                "sv": {
```

Comment:
How do we deal with existing assets - IIUC this is required and will make every json out there fail spec validation?

One idea would be to make it optional with a 1.0 default.

Reply:
Oh, that's right, there's a separate `required` section, nvm.